spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+feature 'comments' do
+
+context 'when signed in' do
```

Comment:
There is a great deal of repetition in these specs. Please consider how you would refactor this. This might help: https://gist.github.com/ptolemybarnes/2dfda607b85d01e113b0